src/components/ui/Navigation.vue

```diff
@@ -24,26 +23,17 @@
         <div v-show="isConnected" class="navbar-item user-identity">
           <JazzIcon
             class="jazzicon"
-            :address="ethAddress" 
```

Comment:
The Jazzicon needs the full Eth address or it will render the wrong icon. `ethAddressString` is just meat for what the user sees.

Reply:
oops